tests/_internal/mbed_tools/test_export.py

```diff
@@ -29,10 +30,14 @@ def test_export(self, mock_write_cmakelists_file, mock_generate_cmakelists_file)
         mock_write_cmakelists_file.assert_called_once_with(
             pathlib.Path(output_dir), mock_generate_cmakelists_file.return_value
         )
+        mock_echo.assert_called_once_with(
```

Comment:
No need to mock, `result.output` should contain whatever is output. I think `self.assertIn` could fit here nicely if you can't do direct equality due to newlines or what not.